Sources/Core/API/TextureManager.swift

```diff
@@ -7,6 +7,13 @@
 import Foundation
 import CoreGraphics
 
+///
```

Comment:
Seems like you forgot to commit the actual comment 🙂

Reply:
Oops yeah. Was working on that. 😓 Committed now.
Will get back to you on the tests.